### PR TITLE
Better handling of redux actions

### PR DESCRIPTION
**Pull request by benfoley**
The previous technique was more concise, but too hard to read. These changes separate the actions into individual files for each step, use constants now for actionTypes, and includes some changes to the step formatting to make it simpler to read.

Oh, it also makes it easier to do things in MapDispatchToProps after actions resolve 🎉 Which was the primary motivation.

### CODE REVIEW

File: src/components/Steps/Dataset/Prepare.js [hunk 1]
```diff
@@ -40,10 +39,10 @@ class DatasetPrepare extends Component {
     }
 
     render() {
-        const { t, list } = this.props
+        const { t, wordlist } = this.props
```
Comment (nicklambourne) [inline src/components/Steps/Dataset/Prepare.js]: What is `t`?

File: src/components/Steps/PronDict/New.js [hunk 2]
```diff
@@ -14,7 +13,7 @@ class PronDictNew extends Component {
     componentDidMount() {}
 
     render() {
-        const { t, name, pronDictNew } = this.props;
+        const { t } = this.props;
```
Comment (nicklambourne) [inline src/components/Steps/PronDict/New.js]: `t`??

File: src/redux/actions/datasetActions.js [hunk 3]
```diff
@@ -0,0 +1,195 @@
+import axios from 'axios'
+import urls from 'urls'
+
+import * as actionTypes from '../actionTypes/datasetActionTypes';
+
+const baseUrl = (process.env.REACT_APP_BASEURL) ? process.env.REACT_APP_BASEURL : 'http://'+window.location.host
+
+
+/* * * * * * * * * * * *  NEW * * * * * * * * * * *  */
```
Comment (nicklambourne) [inline src/redux/actions/datasetActions.js]: god this file looks nice

Comment (benfoley) [reply src/redux/actions/datasetActions.js]: haha, I hope you're not being sarcastic ;-)

File: src/components/App.js [hunk 4]
```diff
@@ -1,14 +1,15 @@
 import React, { Component } from 'react';
 import { BrowserRouter as Router, Route } from "react-router-dom";
 import './App.css'
+// import 'semantic-ui-css/semantic.min.css'
```
[the remark this answers is not in the document]
Comment (benfoley) [reply src/components/App.js]: One day I'll work out how to get good support for semantic ui themes when offline.. this is part of that minor challenge. For now it's in the coming back later pile..

File: public/index.html [hunk 5]
```diff
@@ -6,9 +6,7 @@
         <link rel="shortcut icon" href="%PUBLIC_URL%/favicon.ico" />
         <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no" />
         <meta name="theme-color" content="#000000" />
-        <link rel="manifest" href="%PUBLIC_URL%/manifest.json" />
-        <link rel="stylesheet" href="//cdn.jsdelivr.net/npm/semantic-ui@2.4.1/dist/semantic.min.css" />
-        <!-- <link rel="stylesheet" href="/semantic.css" /> -->
+        <!-- <link rel="stylesheet" href="//cdn.jsdelivr.net/npm/semantic-ui@2.4.1/dist/semantic.min.css" /> -->
```
[the remark this answers is not in the document]
Comment (benfoley) [reply public/index.html]: Part of the offline support for semantic themes

File: src/components/Steps/Dataset/Prepare.js (hunk 1 above)
Comment (benfoley) [reply src/components/Steps/Dataset/Prepare.js]: `t` is shorthand for `react-i18next` library's string translating. Let's mention this in the docs.

File: src/components/Steps/PronDict/New.js (hunk 2 above)
Comment (benfoley) [reply src/components/Steps/PronDict/New.js]: `react-i18next` again..

**Pull request merged**